src/ansi-c/get-gcc-builtins.sh

```diff
@@ -102,7 +102,7 @@ cat > builtins.h <<EOF
 // some newer versions of GCC apparently support __floatXYZ
 #define dfloat32_type_node __float32
 #define dfloat64_type_node __float64
-#define dfloat128_type_node __float128
+#define dfloat128_type_node __CPROVER_Float128
```

Comment:
Randomly placing this comment: the commit message says "bugfix" but it's not clear how this bug has been found/how a future regression is avoided?